packages/lesswrong/server/callbacks/digestCallbacks.tsx

```diff
@@ -20,3 +20,35 @@ getCollectionHooks("Digests").updateAsync.add(async ({document, oldDocument, con
     context
   })
 })
+
+getCollectionHooks("Digests").updateAsync.add(async ({document, oldDocument, context}: {document: DbDigest, oldDocument: DbDigest, context: ResolverContext}) => {
```

Comment:
`document` is a deprecated field. I would use `data`.

Reply:
Ok, though it says to use `newDocument`

Reply:
newDocument is the direct replacement, but I think you want data here.

Reply:
Why?

Reply:
There shouldn't be a difference, but you don't _need_ the combined document. You care about the mutation coming in, not some (slightly hacky) preview of the newDocument